eth-contracts/contracts/DelegateManagerV2.sol

```diff
@@ -0,0 +1,1392 @@
+pragma solidity ^0.5.0;
```

Comment:
consider linking to a hosted diff at https://text-compare.com/ or https://www.diffchecker.com/

Reply:
that would be really great

Reply:
waiting on roneil resp, will handle next week given new timeline